pytorch_optimizer/optimizer/nero.py

```diff
@@ -79,16 +84,20 @@ def step(self, closure: CLOSURE = None) -> LOSS:
 
                 state['step'] += 1
 
+                grad_norm = neuron_norm(grad)
+
+                exp_avg_sq = state['exp_avg_sq']
+                exp_avg_sq.mul_(self.beta).addcmul_(grad_norm, grad_norm, value=1.0 - self.beta)
+
                 bias_correction: float = 1.0 - self.beta ** state['step']
-                state['exp_avg_sq'] = self.beta * state['exp_avg_sq'] + (1.0 - self.beta) * neuron_norm(grad) ** 2
 
-                grad_normed = grad / (state['exp_avg_sq'] / bias_correction).sqrt()
+                grad_normed = grad / (exp_avg_sq / bias_correction).sqrt() + self.eps
```

Comment:
Shouldn't this be 
```
grad_normed = grad / ( (exp_avg_sq / bias_correction).sqrt() + self.eps)
```

Reply:
my mistake thanks for the review :) fixed https://github.com/kozistr/pytorch_optimizer/pull/121/commits/6bd8afcda62e70f3323ed1e062ff9b7be9356bd2